aspnetcore/fundamentals/environments.md

```diff
@@ -1,601 +1,580 @@
 
-> [!WARNING]
-> `launchSettings.json` shouldn't store secrets. The [Secret Manager tool](xref:security/app-secrets) can be used to store secrets for local development.
+* With the project selected in **Solution Explorer**, select **{PROJECT NAME} Debug Properties** from the **Debug** menu, where the `{PROJECT NAME}` placeholder is a project name.
 
-When using [Visual Studio Code](https://code.visualstudio.com/), environment variables can be set in the `.vscode/launch.json` file. The following example sets several [environment variables for Host configuration values](xref:fundamentals/host/web-host#host-configuration-values):
+The preceding approaches open the **Launch Profiles** dialog where you can edit the environment variable settings in the `launchSettings.json` file. Changes made to project profiles may not take effect until the web server is restarted. Kestrel must be restarted before it can detect changes made to its environment.
 
-:::code language="json" source="environments/samples/6.x/EnvironmentsSample/.vscode/launch.json" range="4-10,32-38":::
+Profiles can be selected in the Visual Studio UI next to the Start button (►).
 
-The `.vscode/launch.json` file is used only by Visual Studio Code.
+When a solution contains multiple projects, only set the environment for the startup project.
 
-### Production
+Alternatively, use the [`dotnet run`](/dotnet/core/tools/dotnet-run) command with the [`-lp|--launch-profile` option](/dotnet/core/tools/dotnet-run#options) set to the profile's name. *This approach only supports Kestrel profiles.*
 
-The production environment should be configured to maximize security, [performance](xref:performance/overview), and application robustness. Some common settings that differ from development include:
+```dotnetcli
+dotnet run -lp "https"
+```
 
-* [Caching](xref:performance/caching/memory).
-* Client-side resources are bundled, minified, and potentially served from a CDN.
-* Diagnostic error pages disabled.
-* Friendly error pages enabled.
-* Production [logging](xref:fundamentals/logging/index) and monitoring enabled. For example, using [Application Insights](/azure/application-insights/app-insights-asp-net-core).
+When using [Visual Studio Code](https://code.visualstudio.com/), set the `ASPNETCORE_ENVIRONMENT` environment variable in the `.vscode/launch.json` in the `env` section, along with any other environment variables set in the section:
 
-## Set the environment by setting an environment variable
+```json
+"env": {
+    "ASPNETCORE_ENVIRONMENT": "Staging",
+    ...
+},
+```
 
-It's often useful to set a specific environment for testing with an environment variable or platform setting. If the environment isn't set, it defaults to `Production`, which disables most debugging features. The method for setting the environment depends on the operating system.
+The `.vscode/launch.json` file is only used by Visual Studio Code.
```

Comment:
Using *launch.json* isn't necessary with C# Dev Kit. C# Dev Kit will pick up the available launch profiles from your *launchSettings.json* file.